src/olympia/templates/photon-footer.html

```diff
@@ -20,24 +20,34 @@ <h4 class="Footer-links-header"><a href="{{ url('home') }}">Add-ons</a></h4>
             </ul>
         </section>
 
-        <section class="Footer-firefox-links">
-            <h4 class="Footer-links-header"><a href="https://www.mozilla.org/firefox/">Firefox</a></h4>
+        <section class="Footer-browsers-links">
+            <h4 class="Footer-links-header">Browsers</a></h4>
 
             <ul class="Footer-links">
-                <li><a href="https://www.mozilla.org/firefox/new/?utm_source=addons.mozilla.org&amp;utm_campaign=footer&amp;utm_medium=referral">Download Firefox</a></li>
-                <li><a href="https://www.mozilla.org/firefox/android/?utm_source=addons.mozilla.org&amp;utm_campaign=footer&amp;utm_medium=referral">Android Browser</a></li>
-                <li><a href="https://www.mozilla.org/firefox/ios/?utm_source=addons.mozilla.org&amp;utm_campaign=footer&amp;utm_medium=referral">iOS Browser</a></li>
-                <li><a href="https://www.mozilla.org/firefox/focus/?utm_source=addons.mozilla.org&amp;utm_campaign=footer&amp;utm_medium=referral">Focus Browser</a></li>
-                <li><a href="https://www.mozilla.org/firefox/desktop/?utm_source=addons.mozilla.org&amp;utm_campaign=footer&amp;utm_medium=referral">Desktop Browser</a></li>
-                <li><a href="https://www.mozilla.org/firefox/channel/desktop/?utm_source=addons.mozilla.org&amp;utm_campaign=footer&amp;utm_medium=referral">Beta, Nightly, Developer Edition</a></li>
+                <li><a href="https://www.mozilla.org/firefox/new/?utm_content=footer-link&utm_medium=referral&utm_source=addons.mozilla.org">Desktop</a></li>
+                <li><a href="https://www.mozilla.org/firefox/mobile/?utm_content=footer-link&utm_medium=referral&utm_source=addons.mozilla.org">Mobile</a></li>
```

Comment:
The order of the params is inconsistent across the various links. I.e., some have `utm_content` first and some have `utm_source`, etc. It would be cleaner and easier to read if all of the links had the params in the same order.